reference-architecture/vmware-ansible/roles/static-ipaddress/templates/ifcfg-public.j3

```diff
@@ -0,0 +1,13 @@
+TYPE=Ethernet
```

Comment:
gotcha, I would suggest either a parameterized template or two separate templates (both using the .j2 suffix).